### PR TITLE
Replace deprecated dplyr functions in Readme

### PR DESCRIPTION
**Pull request by abidawson**
When I ran the example code provided in the Readme, I got warnings about deprecated dplyr functions, so I replaced them with the newer versions from dplyr and tibble.

### CODE REVIEW

**Comment by ricardo-bion** [comment]
@abidawson Can you not replace the figure with the PR? Thanks!

**Comment by abidawson** [comment]
@ricardo-bion Sure, sorry about that - set it back to the original now.

File: README.md [hunk 1]
```diff
@@ -57,7 +58,7 @@ This package requires the Airbnb font and the extrafont package.
 
 ```r
 # configured to work on a Mac, change directory to Unix or Windows
-download.file("https://github.com/ricardo-bion/ggtech/blob/master/Circular%20Air-Light%203.46.45%20PM.ttf", "~/Circular Air-Light 3.46.45 PM.ttf", method="curl")
```
Comment (ricardo-bion) [inline README.md]: @ abidawson The fonts are now on Github. Can you fix this last one thing, and then we are ready to merge? Thanks!

Comment (abidawson) [reply README.md]: @ricardo-bion Should be fixed now, if not (or anything else is wrong) let me know. Thanks!

**Comment by ricardo-bion** [comment]
thanks @abidawson!

**Pull request merged**